sdk-lib/src/main/java/cash/z/ecc/android/sdk/internal/transaction/TransactionEncoder.kt

```diff
@@ -25,6 +25,7 @@ internal interface TransactionEncoder {
      */
     suspend fun createTransaction(
         usk: UnifiedSpendingKey,
+        account: Account,
```

Comment:
We should probably look up the account by deriving the UFVK from the USK and then using that to determine the account ID here.

Reply:
To clarify, `account` should not be an additional parameter, because we have enough information to discover it inside of `createSpend`.

Reply:
Filed #1652.